app/workers/tags/alias_retag_worker.rb

```diff
@@ -0,0 +1,18 @@
+module Tags
+  class AliasRetagWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :low_priority, retry: 5
+
+    def perform(tag_id)
+      tag = Tag.find_by(id: tag_id)
+      return unless tag
+
+      tag.taggings.each do |tagging|
+        taggable = tagging.taggable
+        new_tag_list = ActsAsTaggableOn::TagParser.new(taggable.tag_list).parse
```

Comment:
I had briefly considered directly updating the database for performance reason, but decided against it. After digging through our tag code, I found this in `app/lib/acts_as_taggable_on`. which normalizes tags to their preferred names:

```ruby
    def replace_with_tag_alias(tags)
      tags.map do |tag|
        possible_alias = tag
        found_alias = tag
        until possible_alias.nil?
          possible_alias = find_tag_alias(possible_alias)
          found_alias = possible_alias if possible_alias
        end
        found_alias
      end
    end
```

Since we're also using this elsewhere in the app it seemed like the most consistent choice.